website/source/_redirects

```diff
@@ -21,3 +21,6 @@
 /docs/platforms.html                            /docs/builders/index.html
 /intro/platforms.html                           /docs/builders/index.html
 /docs/templates/configuration-templates.html    /docs/templates/engine.html
+/docs/machine-readable/*                        /docs/commands/index.html
+/docs/command-line/*                            /docs/commands/
```

Comment:
These two lines should be changed to:
```
/docs/command-line/*                            /docs/commands/:splat
/docs/extend/*                                  /docs/extending/:splat
```
according to https://www.netlify.com/docs/redirects/#splats